README.md

```diff
@@ -50,18 +50,52 @@ If you just want to get the views for an id and don't want to increment the view
 
 If you want to get all views for all ids, set the `all` query parameter to `true` on a root request. (i.e. `/?all=true`) If you pass the `all` parameter to an id, all ids starting with that pathname will be included. E.g. `/x?all=true` will match views for `/x`, `/xyz` but not `/y`.
 
+### Options
+
+```
+$ micro-analytics --help
+Usage: micro-analytics [options] [command]
+
+Commands:
+
+  help  Display help
+
+Options:
+
+  -a, --adapter [value]  Database adapter used (defaults to "flat-file-db")
+  -h, --help             Output usage information
+  -H, --host [value]     Host to listen on (defaults to "0.0.0.0")
+  -p, --port <n>         Port to listen on (defaults to 3000)
+  -v, --version          Output the version number
+```
+
 ### Database adapters
 
 By default, `micro-analytics` uses `flat-file-db`, a fast in-process flat file database, which makes for easy setup and backups.
 
-This works fine for side-project usage, but for a production application with bajillions of visitors you might want to use a real database with a _database adapter_. Install the necessary npm package (e.g. `micro-analytics-adapter-xyz`) and then specify the `DB_ADAPTER` environment variable: `$ DB_ADAPTER=xyz micro-analytics`
+This works fine for side-project usage, but for a production application with bajillions of visitors you might want to use a real database with a _database adapter_. Install the necessary npm package (e.g. `micro-analytics-adapter-xyz`) and then specify the `DB_ADAPTER` environment variable: `$ DB_ADAPTER=xyz micro-analytics` or use the `--adapter` cli option.
 
 These are the available database adapters, made by the community:
 
 - [`micro-analytics-adapter-redis`](https://github.com/relekang/micro-analytics-adapter-redis)
 
 Don't see your favorite database here? Writing your own adapter is super easy! See [`writing-adapters.md`](writing-adapters.md) for a simple step-by-step guide.
 
+### Live updates
+
+micro-analytics has support for live updates with [server-sent events][]. This is an optional feature
```

Comment:
Rather than being so dry here I'd hype this up a little bit:

`micro-analytics` also let's you listen into updates live with [server-sent events]. That means you can e.g. build a realtime dashboard for your analytics!

> Note: Make sure your database adapter supports this feature. If not, bug them to implement it!